README.md

```diff
@@ -1,3 +1,39 @@
 go-impala is a Go driver for Cloudera Impala
 
 It's based on [impalathing](https://github.com/koblas/impalathing)
+
+## Using
+```go
+package main
+
+import (
+    "log"
+    "fmt"
+    "time"
+    impalathing "github.com/bippio/go-impala"
+)
+
+func main() {
+    host := "<impala-host>"
+    port := 21000
+
+    opts := impalathing.DefaultOptions
+
+    // enable LDAP authentication:
+    opts.UseLDAP = true
+    opts.Username = "<username>"
+    opts.Password = "<password>"
+
+    con, err := impalathing.Connect(host, port, opts)
```

Comment:
should be con, err := impalathing.Connect(host, port, &opts)

Reply:
fixed